ios_sample_test_app/ios_sample_test_app/ViewController.swift

```diff
@@ -0,0 +1,488 @@
+            readDeepLinkButton.widthAnchor.constraint(equalToConstant: view.frame.size.width - 60),
+            readDeepLinkButton.heightAnchor.constraint(equalToConstant: 55),
+            readDeepLinkButton.centerXAnchor.constraint(equalTo: contentView.centerXAnchor),
+
+            readDeepLinkButton.bottomAnchor.constraint(equalTo: contentView.bottomAnchor, constant: 0.0),
+            readDeepLinkButton.trailingAnchor.constraint(equalTo: contentView.trailingAnchor, constant: 0.0),
+        ])
+        
+        setupNavigationMenuButtons()
+        
+        let qrCodeHorizontalConstraint = NSLayoutConstraint(item: qrCode, attribute: NSLayoutConstraint.Attribute.centerX, relatedBy: NSLayoutConstraint.Relation.equal, toItem: self.view, attribute: NSLayoutConstraint.Attribute.centerX, multiplier: 1, constant: 0)
+        let qrCodeVerticalConstraint = NSLayoutConstraint(item: qrCode, attribute: NSLayoutConstraint.Attribute.centerY, relatedBy: NSLayoutConstraint.Relation.equal, toItem: self.view, attribute: NSLayoutConstraint.Attribute.centerY, multiplier: 1, constant: 0)
+        let qrCodeWidthConstraint = NSLayoutConstraint(item: qrCode, attribute: NSLayoutConstraint.Attribute.width, relatedBy: NSLayoutConstraint.Relation.equal, toItem: nil, attribute: NSLayoutConstraint.Attribute.width, multiplier: 1, constant: 350)
+        let qrCodeHeightConstraint = NSLayoutConstraint(item: qrCode, attribute: NSLayoutConstraint.Attribute.height, relatedBy: NSLayoutConstraint.Relation.equal, toItem: nil, attribute: NSLayoutConstraint.Attribute.height, multiplier: 1, constant: 350)
+        
+        self.view.addConstraints([qrCodeHorizontalConstraint, qrCodeVerticalConstraint, qrCodeWidthConstraint, qrCodeHeightConstraint])
+        
+        dimScreen.frame = CGRect(x: 0, y: 0, width: view.frame.size.width, height: view.frame.size.height)
+    }
+    
+    // ---------- Setup Navigation Bar Items ----------
+    
+    func setupNavigationMenuButtons(){
+        let shareButton = UIBarButtonItem(image: UIImage(systemName: "square.and.arrow.up"), style: .plain, target: self, action: #selector(shareBranchLink))
+        
+        let qrCodeButton = UIBarButtonItem(image: UIImage(systemName: "qrcode"), style: .plain, target: self, action: #selector(createQRCode))
+        
+        self.navigationItem.rightBarButtonItems = [shareButton, qrCodeButton]
+    }
+    
+    // ---------- UILabels and UIImages ----------
+    
+    private let generatedBranchLink: UILabel = {
+        let label = UILabel()
+        label.textColor = UIColor(rgb: 0x1F2852)
+        label.numberOfLines = 1
+        label.textAlignment = .center
+        label.font = UIFont(name: "Raleway-Medium", size: 14)
+        label.layer.borderColor = UIColor(rgb: 0xBCC0D4).cgColor
+        label.layer.borderWidth = 1.5
+        label.layer.cornerRadius = 5
+        label.translatesAutoresizingMaskIntoConstraints = false
+        return label
+    }()
```

Comment:
Let's update this from a UILabel to a UITextView so the link can be copied and tapped.

Reply:
Modified to UITextView